webapp/src/main/resources/output.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// var exports = module.exports{}
+function unknownColumnWarning() {
+  // TODO: change from hardcoded to parameters
+  const params = {
+    numNotices: 1,
+    notices: [
+      {
+        filename: "hello.txt",
+        fieldName: "column name",
+        index: 5
+      }
+    ]
+  }
```

Comment:
This should probably be an input parameter

Reply:
Sorry what do you mean by this?